packages/framework/devtool.ts

```diff
@@ -0,0 +1,15 @@
+import { writeFileSync } from 'fs';
+
+import $RefParser from '@apidevtools/json-schema-ref-parser';
+
+const main = async () => {
+  const parser = new $RefParser();
+  const schema = await parser.dereference(__dirname + '/input.json');
+  // Edit this to get correct schema.
```

Comment:
Modifying instructions for clarity

```suggestion
  // Edit this path to target the JSON schema that the send message endpoint uses
```